client/transactions/list/index.tsx

```diff
@@ -371,6 +413,14 @@ export const TransactionsList = (
 				),
 			};
 		};
+		const formatCustomerAmount = () => {
+			return {
+				value: txn.customer_amount / 100,
```

Comment:
This may give an unexpected value for zero-decimal currencies. I think we created a utility function earlier to handle such cases, but I cannot find it at the moment.

Reply:
I couldn't locate such a method within the utils. However, the same division logic is applied in the [`formatAmount` method](https://github.com/Automattic/woocommerce-payments/blob/124c31cc57b20241d0473372fc967a0135d7d4b0/client/transactions/list/index.tsx#L388).

Reply:
In that case, it could be a good opportunity to create a helper function to format amount and refactor `formatAmount` to use it too. See the following links for more context:

pepjwh-il-p2#comment-394
https://github.com/Automattic/woocommerce-payments-server/pull/3549

Reply:
In my opinion, that change falls outside the scope of the current PR. I will create an issue to examine and update all instances where division by 100 is employed. WDYT?

Reply:
A new issues has been created #7834

Reply:
Awesome, thanks! Marking this thread as resolved.